src/components/CourseInfo/CourseInfo.jsx

```diff
@@ -0,0 +1,40 @@
+import { Link, useParams } from 'react-router-dom';
+
+import Info from '../../common/Info/Info';
+import { getCourseInfo } from '../../helpers/getCourseInfo';
+
+import { mockedCoursesList } from '../../constants';
+import './CourseInfo.css';
+
+export default function CourseInfo() {
+	const { courseId } = useParams();
+	const course = mockedCoursesList.filter(
+		(course) => course.id === courseId
+	)[0];
+	return (
+		<div className='main-container'>
+			<Link className='link' to='/courses'>
+				&larr; Back to courses
+			</Link>
+			<h2>{course.title}</h2>
+			<div className='courseinfo'>
+				<span className='description'>{course.description}</span>
+				<div className='information'>
+					{getCourseInfo(course, ['id', 'duration', 'created']).map(
```

Comment:
Let's try to grab these hardcoded params from course itself, you can do this via Object.keys. then filter them to remove unnecessary (here it is title and description). Save it in a variable which you can use here

Reply:
It's quite efficient. Thanks!

Reply:
Well, after thinking about it for a moment, I decided it would be much easier to just put there an array of desired params instead of filtering from course. I'm talking about that specially in the context of CourseCard. the situation there is somewhat different (the course info is taken through props instead of course itself). 
Thus I will leave it as it is, but if you have a better idea (or maybe I didn't understand your proposition), I will change it :)